src/main/frontend/src/routes/context.svelte

```diff
@@ -22,9 +27,16 @@
 </svelte:head>
 
 <!-- add newbie tooltips -->
-<CardForm title="Context" next="aggregates" bind:valid>
-	<TextField class="mb-4 pb-4" placeholder="com.example" bind:value={groupId} rules={[requireRule, packageRule]} validateOnBlur={!groupId}>Group Id</TextField>
-	<TextField class="mb-4 pb-4" placeholder="demo" bind:value={artifactId} rules={[requireRule, artifactRule]} validateOnBlur={!artifactId}>Artifact Id</TextField>
-	<TextField class="mb-4 pb-4" placeholder="1.0.0" bind:value={artifactVersion} rules={[requireRule, versionRule]} validateOnBlur={!artifactVersion}>Artifact Version</TextField>
-	<TextField class="mb-4 pb-4" placeholder="com.example.demo" bind:value={packageName} rules={[requireRule, packageRule]} validateOnBlur={!packageName}>Base Package Name</TextField>
+<CardForm title="Context" previous="platform" next="aggregates" bind:valid>
+	{#if $platformSettings && $platformSettings.platform === '.NET'}
+		<TextField class="mb-4 pb-4" bind:value={projectName} rules={[requireRule]}>Project Name</TextField>
```

Comment:
Might be useful to allow only valid names according to conventions: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-assemblies-and-dlls

Reply:
Also please add it as a 2nd filed on the form